README.md

```diff
@@ -598,15 +598,16 @@ row and third column would be the age of the second person in our data.
 The second row as a whole would be all the data for that second person,
 i.e. the height, weight and age of that person.  
 
-**Note that that row would also be cnsidered a vector.  The third column
+**Note that that row would also be considered a vector. The third column
 as a whole would be the vector of all ages in our dataset.**
 
 As our first example, consider the **ToothGrowth** dataset built-in to
 R.  Again, you can read about it in the online help by typing
 
 ``` r
 > ?ToothGrowth
-``` (The data turn out to be on guinea pigs, with orange juice or
+```
+(The data turn out to be on guinea pigs, with orange juice or
```

Comment:
Note was written on the same line as previous code fence so it didn't appear as distinct line